share/lang/cn.json

```diff
@@ -285,6 +285,5 @@
       "2b9": "次元裂缝零式欧米茄：德尔塔之章3",
       "2ba": "次元裂缝零式欧米茄：德尔塔之章4"
     }
-  },
-  "_author": "Chihaya Gunzo"
+  }
```

Comment:
`_author` field exists as metadata and reserved for future use, please keep this.

Reply:
OK